recipe/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set reponame = "mpl-stylesheets" %}
```

Comment:
please check if it is usual to put the conda recipe in the repo of the actual package.  I am not sure.

Reply:
Oh I see. You are right. conda recipe would be in a fork of conda-forge repo. Thanks Simon!